### ISSUE TITLE
Cant create: multiple projects with identical repo name and different repo urls

### ISSUE THREAD

**Issue by kokare-darshan** [alefragnani/vscode-project-manager#718]
### Discussed in https://github.com/alefragnani/vscode-project-manager/discussions/717

<div type='discussions-op-text'>

<sup>Originally posted by **kokare-darshan** December 14, 2023</sup>
Hi guys,

Lets say I have a repo-A and I add it to project-manager. Now I fork repo-A and lets say its under folder "user-name/repo-A", how can I add new repository which has same name as earlier repo? I see that it updates same repo link today.

tldr: I want 2 projects with same name.</div>

**Comment by alefragnani** [comment]
Hi @kokare-darshan ,

On Favorite Projects (the ones you hit `Save Project`) yes, you can't have duplicates. The name works as its identifier, and if you try to reuse the name on _another path_ it will ask you if you want to replace its path. 

In order to have both projects, you must change the second one, maybe adding a prefix/suffix, so you will be able to have both
